### ISSUE TITLE
Missing syntax error for "ru" (raw Unicode) string prefix

### ISSUE THREAD

**Issue by correctmost** [microsoft/pyright#10222]
**Describe the bug**

Pyright does not emit any errors when "ru" is used as a string prefix.

Python 2.7 and some Python 3.3 alphas [accepted this syntax](https://github.com/python/cpython/issues/59301#issuecomment-1093582901), but newer versions [do not](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals).

mypy, ruff, and Pylint warn about syntax errors in the following code:

**Code or Screenshots**

```python
x = ru'1'
```

**VS Code extension or command-line**
Command-line version 1.1.398

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.399.
